app/javascript/packs/delete_button.js

```diff
@@ -0,0 +1,23 @@
+$('#input').change(function () {
+    button()
+})
+
+$('#button').click(function () {
+    $('#input').val('')
+    let elem = document.getElementById('your_preview_id')
+    while(elem.attributes.length > 1)
+        elem.removeAttribute(elem.attributes[1].name);
+    button()
+})
+
+button()
+
+function button() {
```

Comment:
use arrow functions, they are shorter, and use more understandable variable and function names, I don't understand what does 'button' function should do